Composer/packages/server/src/models/bot/builder.ts

```diff
@@ -160,7 +160,7 @@ export class Builder {
     if (!luFiles.filter((file) => !emptyFiles[file.name]).length) return;
 
     const nlrList = await this.runOrchestratorNlrList();
-    const defaultNLR = nlrList.default;
+    const defaultNLR = nlrList.defaults.en_intent;
```

Comment:
I wonder if we should add a TODO here to hoist this hardcoded default into something configurable.. What is the user behavior? When would they _not_ want to default to English here?

Reply:
I broke this change apart from upcoming Orchestrator multilang changes to keep this PR small and focused on the SDK/CLI name changes, and just maintain parity with what we currently have.

The multilang PR will hopefully address this concern of defaulting to English only when there are English locale LU files.